.github/workflows/axon-alphanet-test.yaml

```
@@ -0,0 +1,22 @@
+name: Migrate Contract to Axon-Alphanet
+
+on:
+  push:
```

```suggestion
  workflow_dispatch:
```

It supports triggering this workflow manually.